pyvisfile/vtk/vtk_ordering.py

```diff
@@ -182,25 +182,24 @@ def vtk_lagrange_tetrahedron_node_tuples(order):
     return nodes
 
 
-def vtk_lagrange_simplex_node_tuples(dims, order, is_consistent=False):
+def vtk_lagrange_simplex_node_tuples(dims, order, vtk_version=(2, 1),
+        is_consistent=None):
     """
     :arg dims: dimension of the simplex, i.e. 1 corresponds to a curve, 2 to
         a triangle, etc.
     :arg order: order of the polynomial representation, which also defines
         the number of nodes on the simplex.
-    :arg is_consistent: If *True*, 1D curve node ordering will follow the
-        same rules as the higher-dimensional simplices by putting the
-        vertices first. This is not the default in VTK as of version 8.1,
-        when the higher-order elements were introduced.
+    :arg vtk_version: a :class:`tuple` of two elements containing the version
```

Comment:
I left the `vtk_version` here, even though it's no necessary anymore for simplices, for consistency with the quad routine and in case they change things in the future.